### DISCUSSION TITLE
Umlauts in HTML metadata

### DISCUSSION THREAD

**Discussion by rockgobo** [2sic/2sxc#3225] [Q&A]
This may be a very german problem, but i wanted to fill in the metadata tags of the generated HTML with something meaningfull and realised that 2sxc HtmlPage.Description = "ä" and Kit.Page.SetDescription("ä") both end up in "&#228;". Our problem is that these Characters are not interpreted in Tools like Meltwater, that use these metadata to fill in Textblocks in Newsletters.

Is there a way to set descriptions that are not encoded and use standard utf8 texts?

**Comment by iJungleboy** [comment]
I don't know what code you have to create this, but I presume that if you pre-encode it it may work...

it could also be a "bug" in a way that it's a bit too careful - to be honest UTF8 is the default today so it wouldn't need that.

Note that I would scrable the screenshot as it contains your e-mail address.

**Comment by rockgobo** [comment]
![image](https://github.com/2sic/2sxc/assets/3286890/cde20585-e27e-4e7c-82be-5c431ff40abb)

The Code (2sxc 16.0) is very simple and DNN (9.12) is generating the webside with utf-8:

![image](https://github.com/2sic/2sxc/assets/3286890/cfb27ff3-3cb9-4697-ba6f-66ebb80f89c8)

Thank you for the hind, i deleted my first screenshot.

**Comment by iJungleboy** [comment]
Just a question: I notice you are using an old API which isn't maintained any more - the `HtmlPage.Description`.

Could you try the new API using the `IPageService`? Assuming you're using Razor14, it would be on `Kit.Page.SetDescription(...)` or on older Razor, something like

```c#
var page = GetService<ToSic.Sxc.Services.IPageService>();
page.SetDescription(...);
```

See https://docs.2sxc.org/api/dot-net/ToSic.Sxc.Services.IPageService.html#ToSic_Sxc_Services_IPageService_SetDescription_System_String_System_String_

**Comment by rockgobo** [comment]
I will try to rewrite everything.
